pulsar-functions/worker/src/test/java/org/apache/pulsar/functions/worker/rest/api/v2/FunctionApiV2ResourceTest.java

```diff
@@ -210,7 +216,7 @@ public void testRegisterFunctionMissingNamespace() throws IOException {
                 "Namespace is not provided");
```

Comment:
We don't need to specify the error message here is its already specified in the annotation

Reply:
Done